libs/sdk-ui-dashboard/src/index.ts

```diff
@@ -26,3 +26,5 @@ export {
     IDashboardKpiCustomizer,
     DashboardStateChangeCallback,
 } from "./plugins/customizer";
+export { InsightPlaceholderWidget } from "./widgets/placeholders/types";
```

Comment:
Nitpick: two exports from the same location.

Reply:
fixed